src/se/vidstige/jadb/managers/PackageManager.java

```diff
@@ -48,8 +48,10 @@ private void verifyOperation(String operation, String target, String result) thr
     }
 
     public void remove(RemoteFile file) throws IOException, JadbException {
-        InputStream s = device.executeShell("rm", "-f", Bash.quote(file.getPath()));
-        Stream.readAll(s, Charset.forName("UTF-8"));
+        try (InputStream stream = device.executeShell("rm", "-f", Bash.quote(file.getPath()))) {
+            Stream.flushRead(stream);
+            stream.close();
```

Comment:
Close should not be needed here as you use try-with-resource? What's wring with the `readAll` method? I feel the name is clearer.